src/runtime/composables.ts

```diff
@@ -1,15 +1,17 @@
+import type { H3Event } from 'h3'
 import type { $Img } from '../module'
 
 import { createImage } from './image'
 import { imageOptions } from '#build/image-options.mjs'
 import { useNuxtApp, useRuntimeConfig } from '#imports'
 
-export const useImage = (): $Img => {
+export const useImage = (event?: H3Event): $Img => {
   const config = useRuntimeConfig()
   const nuxtApp = useNuxtApp()
 
   return nuxtApp.$img as $Img || nuxtApp._img || (nuxtApp._img = createImage({
     ...imageOptions,
+    event: event || nuxtApp.ssrContext?.event,
```

Comment:
This looks a little bit confusing. If we do expect `nuxtApp` context to be available (L10), and assuming nuxt app always has an event in its context, why it has to be passed to `useImage`?